app/views/layouts/_footer.html.erb

```diff
@@ -1,7 +1,7 @@
 <footer class='footer navbar navbar-default navbar-fixed-bottom'>
   <div class='container'>
-    <% if Rails.env.development? %>
-      <%= simple_form_for User.find(1), :html => { :class => 'form-inline' } do |f| %>
+    <% if Rails.env.development? && user_signed_in? %>
```

Comment:
Why this check? In dev (once we have seeds for all common testing scenarios (namely, a teacher)), Google login should be relatively unnecessary, but this requires that we first do a Google login for no reason (AFAICT).